pom.xml

```diff
@@ -4,24 +4,20 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.mwo.group2</groupId>
```

Comment:
ja bym to zmienil na pl.mwo2017.dziekanat albo pl.edu.agh.mwo2017.dziekanat , w sensie to powinno byc tak jak jest powiedziane: FQDN odwrotnie :]